CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Build
+The declaration files are auto generated(as new Corvid apis are added all the time) 
```

Comment:
- Use `autogenerated` or `auto-generated`.
- Change to Corvid APIs
- You can remove the parenthesis and add a period.